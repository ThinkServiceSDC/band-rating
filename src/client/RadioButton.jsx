import React from "react";

export const RadioButton = (props) => {
    return (
        <div className='radio-button'>
            <input type='radio' id={props.text} name='rating'/>
            <label className='radio-button-label'>{props.text}</label>
        </div>
    )
};