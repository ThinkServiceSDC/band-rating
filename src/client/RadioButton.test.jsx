import React from 'react';
import toJSON from 'enzyme-to-json';
import {shallow} from 'enzyme';
import {RadioButton} from "./RadioButton";

describe('RadioButton', () => {
    it('should render correctly', () => {
        expect(toJSON(shallow(<RadioButton text='Good'/>))).toMatchSnapshot();
    });
});