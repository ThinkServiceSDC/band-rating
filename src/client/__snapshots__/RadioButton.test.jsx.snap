// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton should render correctly 1`] = `
<div
  className="radio-button"
>
  <input
    id="Good"
    name="rating"
    type="radio"
  />
  <label
    className="radio-button-label"
  >
    Good
  </label>
</div>
`;
